Makefile

```diff
@@ -132,10 +197,12 @@ scala-artifact = \
 	renga-graph
```

Comment:
TODO: remove the scala artifacts and update readme to remove sbt as a req.

NB: `renku-storage` cab be built with docker only. Also, cascading changes from `renku-commons` is not possible from the docker build so building it here has no value.